lib/coll.gi

```diff
@@ -169,8 +169,11 @@ InstallMethod( IsWholeFamily,
 ##
 #M  Size( <C> ) . . . . . . . . . . . . . . . . . . . .  size of a collection
 ##
-InstallImmediateMethod( Size,
-    IsCollection and HasIsFinite and IsAttributeStoringRep, 0,
+#  replace immediate method by ordinary, as immediate gets called for every
+#  group that knows it is finite but does not know its size -- e.g.
+#  permutation, pc.  The benefit of this is minimal beyond showing off a
+#  feature.
```

Comment:
This comment (as well as several other that follow) only make full sense when read in the context of this commit resp. PR; but if somebody reads this in a year, they might have a hard time understanding what is meant here. "What is being replaced here???"

Not having this commit would IMHO be better; or start it with something like "the following method used to be immediate. However..." but I wonder whether this would be a useful comment. I think this would better fit into the commit message.

Reply:
I'll change the text. IF it it hidden in a commit message, however I think it is less likely to be seen and also loses the reference to the place in the source code.